plugin-gestures/src/main/java/com/mapbox/maps/plugin/gestures/GesturesPluginImpl.kt

```diff
@@ -1393,7 +1393,7 @@ class GesturesPluginImpl : GesturesPlugin, GesturesSettingsBase, MapStyleObserve
     // Prevent drag start in area around horizon to avoid sharp map movements
     val topMapMargin = 0.04 * mapTransformDelegate.getSize().height
     val reprojectErrorMargin = min(10.0, topMapMargin / 2)
-    val point = ScreenCoordinate(pixel.x, pixel.y - topMapMargin)
+    val point = ScreenCoordinate(pixel.x, pixel.y - topMapMargin).sanitizeNaN()
```

Comment:
Seems like we're hiding the actual error here, with either `topMapMargin` or `pixel` holding the original NaN.
Maybe sanitize pixel / margin independently, will at least log the wrong one?

Reply:
from stacktrace I'm seeing that both values from ScreenCoordinate are NaN